.github/ISSUE_TEMPLATE/epic.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Epic
+about: A group of related work items working towards a shared goal. Contains a list of linked Stories.
+title: ''
```

Comment:
Not sure what to have as a title placeholder for Epics.

Reply:
I think it's really just the feature. It took be "Adds inventory enhancements" "Updates inventory enhancements. etc.  so 'Adds [feature] enhancements' then the stories focus more on details?